boards/shields/wnc_m14a2a/boards/nrf52840_pca10056.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_GPIO_NRF_P1=y
```

Comment:
Rather do it as Kconfig.defconfig than .conf direct setting style

Reply:
Like I said above, this particular setting can be abstracted under `CONFIG_ARDUINO_SERIAL` and handled in the board's Kconfig.defconfig.  The nRF52840's don't have the same problem as K64F because they can set any pin to serial.   The tricky part here is that these settings in the .overlay are unique to this shield:
```
	tx-pin = <46>;
	rx-pin = <45>;
	rts-pin = <44>;
	cts-pin = <47>;
```

The standard Arduino serial pins are different.